blog.json

```diff
@@ -20,5 +20,61 @@
+  },
+  {
+    "uuid": "4ac4f8ba-9df9-4874-bcbc-8ac1c5bb0d40",
+    "slug": "what-happens-if-my-mentor-disappears",
+    "category": "updates",
+    "published_at": "2019-01-18 10:12",
+    "content_repository": "blog",
+    "content_filepath": "posts/what-happens-if-my-mentor-disappears.md",
+    "title": "What happens if my mentor disappears?",
+    "author_handle": "iHiD",
+    "marketing_copy": "Sometimes you're working away improving an exercise and your mentor disappers. Up until now, that's been a big problem but this week we've made some big strides forward. Learn more about how we're dealing with it moving forward."
+  },
+  {
+    "uuid": "c4bb5f31-aa09-4b85-8b8f-d2f1125a8549",
+    "slug": "interview-with-erik-schierboom",
+    "category": "interviews",
+    "published_at": "2019-01-21 01:00",
+    "content_repository": "blog",
+    "content_filepath": "posts/interview-with-erik-schierboom.md",
+    "title": "An interview with Erik Schierboom",
+    "author_handle": "iHiD",
+    "marketing_copy": "We talk to Erik Schierboom about functional programming, maintaining the C#, F# and Scala tracks, and our plans for automated static analysis of Exercism solutions."
+  },
+  {
+    "uuid": "8db67471-696c-487a-8bb4-91c239b73166",
+    "slug": "introducing-community-comments",
+    "category": "updates",
+    "published_at": "2019-01-26 01:00",
+    "content_repository": "blog",
+    "content_filepath": "posts/introducing-community-comments.md",
+    "title": "We've relaunched community comments",
+    "author_handle": "iHiD",
+    "marketing_copy": "One consequence of adding mentoring to Exercism last year was the removal of casual commenting. We've now added that back. Read on to learn more."
+  },
+  {
+    "uuid": "f2718cd1-e820-49ce-924b-867e87dd9912",
+    "slug": "interview-with-ryan-potts",
+    "category": "interviews",
+    "published_at": "2019-01-30 22:20",
+    "content_repository": "blog",
+    "content_filepath": "posts/interview-with-ryan-potts.md",
+    "title": "An interview with Ryan Potts",
+    "author_handle": "iHiD",
+    "marketing_copy": "Ryan Potts talks to us about how he got into Delphi, the things he's learnt from being a contributor to Exercism, and his experiences of programming in industries that are still offline."
+  },
+  {
+    "uuid": "e6f3da5f-98b9-4cf7-be0b-447890aff367",
+    "slug": "coding-intentionally-in-bash-grains",
+    "category": "programming skills",
+    "language": "bash",
+    "published_at": "2019-01-18 14:18",
+    "content_repository": "blog",
+    "content_filepath": "posts/coding-intentionally-in-bash-grains.md",
+    "title": "Coding Intentionally in Bash Grains",
+    "author_handle": "rpalo",
+    "marketing_copy": "Often, there are many approaches that all solve the same problem.  Coming up with one that communicates your thought process and goals to others is the hard part."
```

Comment:
It looks like we've got some weirdness in the merging from master.

@rpalo Would you mind doing a rebase onto the most recent master and force push to get rid of the unrelated changes?